client/tokenized-express-checkout/__tests__/tokenized-express-checkout--product-page.test.js

```diff
@@ -44,22 +56,22 @@ describe( 'Tokenized Express Checkout Element - Product page logic', () => {
 			total: { amount: 1100 },
 			shippingOptions: {
 				id: 'pending',
-				label: 'Pending',
+				label: 'Pending server-side',
 				detail: '',
 				amount: 0,
 			},
 			displayItems: [
 				{
-					label: 'Beanie',
+					label: 'Beanie server-side',
```

Comment:
I changed the naming of these labels to (hopefully) make it clearer to understand when a product is provided from the server-side hydration, instead of being provided from the Store API 🤷